setup.py

```diff
@@ -51,6 +51,7 @@
     ],
     extras_require={
         'deep-learning': [
+            'tensorflow',
             'torch',
             'transformers<=2.0.0',
             'spacy-transformers @ https://github.com/nsorros/spacy-transformers/tarball/master#egg=spacy-transformers',
```

Comment:
I'm a bit confused - I thought this dependency on the fork had gone away with the last merge to master. Perhaps you should rebase?

Reply:
I am not changing this line so whatever is in master will change. I can rebase though so that this propagates here, no prob.